website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -0,0 +1,49 @@
+---
+page_title: Destruction and Deletion - Workspaces - Terraform Cloud
+description: |-
+  Learn about destroying infrastructure and deleting workspaces in Terraform Cloud.
+---
+
+# Destruction and Deletion
+
+Terraform Cloud workspaces have two primary delete actions:
+
+- [Destroy Infrastructure](#destroy-infrastructure) deletes resources managed by the Terraform Cloud workspace by triggering a destroy run.
+- [Delete Workspace](#delete-workspaces) deletes the workspace itself without triggering a destroy run.
+
+In general, you should perform both actions in the above order when destroying a workspace to ensure resource cleanup for all infrastructure managed by a workspace.
+
+## Destroy Infrastructure
+
+Destroy plans delete the infrastructure managed by a workspace. It's recommended that you destroy the infrastructure managed by a workspace prior to deleting the workspace itself, otherwise the infrastructure resources will continue to exist but will become unmanaged.
+You will need to go into your infrastructure providers to delete the resources manually.
+
+Destroy plans
+
+Before queuing a destroy plan, you must enable the `Allow destroy plans` setting.
+
+### Manually Destroy
+
+Adds a destroy plan to the run queue.
+
+### Automatically Destroy
+
+<!-- BEGIN: TFC:only name:pnp-callout -->
+@include 'tfc-package-callouts/ephemeral-workspaces.mdx'
```

Comment:
We don't use the term "ephemeral workspaces" anywhere in this article? Should we?

Reply:
I've made the vote for avoiding public language about "ephemeral workspaces", but we already put it in the marketing https://www.hashicorp.com/blog/terraform-ephemeral-workspaces-public-beta-now-available

Reply:
Marketing always loves naming things 😂  The language is also in the partial that we use on this page, so we should probably use the name _somewhere_ on this page to avoid confusion.

Reply:
I can throw it here, but this almost feels like it's repeating the content of the callout 

![image](https://github.com/hashicorp/terraform-docs-common/assets/33099255/9233c79a-5d32-412b-8f4a-69dd92a00f92)

Reply:
...I still like it though 😂

Reply:
Yeah if it helps then that's great!